source/arriving/setup_computer.rst

```diff
@@ -1,51 +1,167 @@
+
+    export WORKON_HOME=~/Envs/
+    #export VIRTUALENVWRAPPER_PYTHON=/usr/bin/python3   # uncomment this line if you have errors when starting your terminal (next step)
+    source /usr/local/bin/virtualenvwrapper.sh
+
+Finally, you can restart the terminal and create an environment:
+
+.. code-block:: bash
+
+    NAME=somename # Ex: $NAME could be my_env_py3
+                    # depending on the use you will give to this environment.
+                    # Try to be explicit
+    # One of:
+    mkvirtualenv $NAME
+    mkvirtualenv --python=python3.10 $NAME   # To use one precise python version
+                                                # You can also use the complete path to the python installation
+    mkvirtualenv $NAME --python=/usr/bin/python3.10  # Equivalent. The python
+                                                        # version must be already
+                                                        # installed on your computer
+
+    # Note. To download a python version to your /usr/bin without installing it:
+    # https://www.python.org/downloads/
+
+When everything is set, restart your terminal. You should now be able to work in a chosen environmnent by using :bash:`workon $NAME`. Now everytime you need to install a new library inside the environment, you should always try to use :bash:`pip install your_library`. The environment will use the right pip based on the current python version. *Don't use sudo pip!* It will use the pip of the system! If the library can't be installed via pip, however, you can use :bash:`sudo apt-get install your_library`. In this case the `sudo` will not be a problem.
 
-    If you work from home, you might need to connect to the UdeS network. Do the following:
+You might want to always have a specific environment when opening a terminal. To do so, simply open the .bashrc and copy-paste this :bash:`workon somename` at the end of it (don't forget to replace :bash:`somename` by the actual name of your environment). Before moving on to the next steps, make sure to have a virtual environment active!
 
-    1) To have access to scientific papers (ex, free access to many articles in Google scholar): See the VPN information below or go on the University's `library's website <https://www.usherbrooke.ca/biblio/trouver-des/articles-de-periodiques-revues-et-journaux/>`_ and click on "Outil de découverte" if your are logged in with your CIP (top-right corner, the connexion button).
+Git
+"""
 
-    2) To connect to your lab computer: Use ssh or TeamViewer (see below).
+Git is a version control software system often used for versioning of source code by developers. In the lab, we use the website `Github <https://github.com/>`_ to help us store and manage our code using Git in a user-friendly way. You should get used to it from the beginning, as it is a powerful and very helpful tool. If you have never used Git on your computer, the first step would be to set your user.name and user.email:
```

Comment:
"allows to keep track of the changes made to a code base." or something?

Reply:
When I explain it to non-scientists, I say it's like dropbox, but for code, and before syncing the differences you made in a file, the changes need to be approved by others.

Reply:
That last bit is only true for a "group project" like scilpy, but if you use github for your own personal code it's even more like dropbox!

Reply:
I changed the text a bit!